components/google_search_console/common/methods.mjs

```
@@ -0,0 +1,309 @@
+    let warnings = [];
+;
+    // Trim the input (already checked for string);
+    const trimmedInput = input.trim();
+
+    // Reject if spaces, tabs, or newlines are present
+    if ((/[ \t\n]/.test(trimmedInput))) {
+      this.throwCustomError( `Url contains invalid characters like space, backslash etc., please check.`  +  this._reasonMsg(reason))
+    };
+  
+  // Warn about suspicious characters
+    const dubiousCharRegex = /[\\<>{}|"`^]|[\u200B\u200C\u200D\u2060\uFEFF\u00A0]/g;
```

Biome (1.9.4)

[error] A character class cannot match a joined character sequence.

A zero width joiner composes several emojis into a new one. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)